<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OkButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="mainMenu">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>351</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="mainMenuLayout">
      <item>
       <widget class="QGroupBox" name="singleMatrixMenu">
        <property name="title">
         <string>Операции над одной матрицей</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>34</x>
           <y>30</y>
           <width>281</width>
           <height>103</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="singleMatrixMenuLayout">
          <item>
           <widget class="QRadioButton" name="newMatrixButton">
            <property name="text">
             <string>Создать новую пустую матрицу</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="itemChangeButton">
            <property name="text">
             <string>Изменить значение элемента матрицы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="getValueButton">
            <property name="text">
             <string>Получение значения элемента матрицы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lineEntryButton">
            <property name="text">
             <string>Построчныый ввод матрицы</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="twoMatrixMenue">
        <property name="title">
         <string>Операции с двумя матрицами</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>261</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="twoMatrixMenueLayout">
          <item>
           <widget class="QGroupBox" name="matrixChangeMenue">
            <property name="title">
             <string>Задание матриц</string>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>39</x>
               <y>20</y>
               <width>204</width>
               <height>103</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="matrixChangeMenueLayout">
              <item>
               <widget class="QRadioButton" name="lineEntryAButton">
                <property name="text">
                 <string>Построчный ввод матрицы А</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="lineEntryBButton">
                <property name="text">
                 <string>Построчный ввод матрицы B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="itemChangeAButton">
                <property name="text">
                 <string>Изменить элемент матрицы A</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="itemChangeBButton">
                <property name="text">
                 <string>Изменить элемент матрицы В</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Разность матриц</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_6">
            <property name="text">
             <string>Проверка равенства матриц</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="saveMatrixButton">
        <property name="text">
         <string>Загрузить сохраненную матрицу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="downloadMatrixButton">
        <property name="text">
         <string>Сохраненную матрицу</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSpinBox" name="sizeSpinBox_1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="AMatrixlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BMatrixLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeSpinBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUTM">
    <property name="title">
     <string>UTM</string>
    </property>
   </widget>
   <addaction name="menuUTM"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
